doc/source/drivers/raster/vrt_processed_dataset.rst

```diff
@@ -132,6 +130,48 @@ The value of ``Algorithm`` must be a registered VRTProcessedDataset function. At
 
 A ``Step`` will generally have one or several ``Argument`` child elements, some of them being required, others optional. Consult the documentation of each algorithm.
 
+Starting with GDAL 3.11, a ``OutputBands`` element can be
+defined as a child element of ``VRTDataset``, with the following 2 attributes:
+
+* ``count`` whose value can be ``FROM_SOURCE`` to indicate that the output band
+  count must be the same as the number of bands of the input dataset,
+  ``FROM_LAST_STEP`` to indicate that it must be the number of output bands
+  returned by the initialization function of the last step, or an integer value.
+
+* ``type`` whose value can be ``FROM_SOURCE`` to indicate that the output band
```

Comment:
I know I suggested `type`, but do you think this should be `dataType` for consistency with other tags?

Reply:
yes, I had also the same thought. Changed to dataType